### DISCUSSION TITLE
Can't compute hex code

### DISCUSSION THREAD

**Discussion by OpenK54** [balu100/dolby-vision-for-windows#5] [Q&A]
You write that one should compute the hex code with "python.exe enable_dolby_vision_hdmi.py __HEX_STRING__". I don't now what do with this line. I cloned the repo with git and installed phyton on my windows pc - no music. So what am I missing? Thanks for help.

**Comment by balu100** [comment]
Did you follow the previous steps? at that point you should have you screens payload (hex string)
use the script like this python.exe enable_dolby_vision_hdmi.py YOUR_HEX_STRING_HERE

**Comment by marZey87** [comment]
Was there any clarity on this? Trying to calculate the HEX code for LG CX but not sure if it's working, what am i missing?
![image](https://github.com/user-attachments/assets/ab02c8c8-8660-4f88-985a-0083d0445ce4)

**Reply by marZey87** [reply]
Figured it out, had to put the command in powershell + full directory location of the .py file
